resources/test_data.json

```diff
@@ -467,5 +467,93 @@
         "note": "",
         "bug_type": "crash",
         "checker_qual_required": false
+    },
+    {
+        "issue_id" : "cf-3025",
+        "url": "https://github.com/tahiat/guava.git",
+        "branch": "cf-3020",
```

Comment:
This looks wrong to me.

Reply:
both the issue has same source code.  i will made a separate fork for this issue id
https://github.com/typetools/checker-framework/issues/3025. --> Professor, the issue reproduces but the stack trace is different that the one attached in the issue. 
I need to know the exact javac version of this issue